### PR TITLE
Shim for HTML Main Tag in IE Added to Layoutslidepanel

### PR DESCRIPTION
**Pull request by amichaelparker**
### Summary
Adds `document.createElement('main')` to resolve error in IE regarding the use of the <main> tag. 

Closes [#351](https://github.com/cerner/terra-framework/issues/351)

### CODE REVIEW

File: packages/terra-layout/src/_LayoutSlidePanel.jsx [hunk 1]
```diff
@@ -68,6 +68,7 @@ class LayoutSlidePanel extends React.Component {
   }
 
   componentDidMount() {
+    document.createElement('main'); // shim for <main> tag not being recognized in IE
```
Comment (StephenEsser) [inline packages/terra-layout/src/_LayoutSlidePanel.jsx]: How does this work? Is it just magic?

Comment (bjankord) [reply packages/terra-layout/src/_LayoutSlidePanel.jsx]: https://weblog.west-wind.com/posts/2015/Jan/12/main-HTML5-Tag-not-working-in-Internet-Explorer-91011

Reading through that link, I think we may be able to drop the JS part and just keep the CSS setting `main { display: block}` I think the element creation may only be needed for IE8 and below is not included in our browser support list.

Comment (amichaelparker) [reply packages/terra-layout/src/_LayoutSlidePanel.jsx]: I'll check it out.

Comment (amichaelparker) [reply packages/terra-layout/src/_LayoutSlidePanel.jsx]: image

This is what I get in IE11 if I remove the JS line.

Comment (bjankord) [reply packages/terra-layout/src/_LayoutSlidePanel.jsx]: ah, good testing. Ok, let's leave the script.

**Pull request merged**